mahta/CSS/CSS responsive design/styles.css

```diff
@@ -0,0 +1,40 @@
+ @media only screen and (max-width: 600px) and (orientation: landscape) {
+   body{
+      background-color: lightgoldenrodyellow;
+   }
+   .Title {
+      text-align:center;
+      font-size: 2rem;
+   }
+      .Subtitle {
+      text-align: center;
+      font-size: 1.25rem;
+   }
+ }
+ @media only screen and (max-width: 600px) {
+   body{
+      background-color: lightgreen;
+   }
+   .Title {
+      text-align:start;
+      font-size: 2rem;
+   }
+      .Subtitle {
+      text-align: right;
+      font-size: 1.25rem;
+   }
+ }
+ @media only screen and (min-width: 600px) {
+   body{
+      background-color: lightblue;
+   } 
+    .Title {
+      text-align: center;
```

Comment:
By middle of the page, I mean aligning both horizontally and vertically.

Reply:
I'm not certain if both align-items & justify-content properties were necessary to add, but I wasn't certain which one to use.